tools/cdi-source-update-poller/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+
+	cdiClientset "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned"
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+)
+
+var (
+	configPath    string
+	kubeURL       string
+	cronNamespace string
+	cronName      string
+	url           string
+)
+
+func init() {
+	flag.StringVar(&configPath, "kubeconfig", os.Getenv("KUBECONFIG"), "(Optional) Overrides $KUBECONFIG.")
+	flag.StringVar(&kubeURL, "server", "", "(Optional) URL address of a remote api server.  Do not set for local clusters.")
+	flag.StringVar(&cronNamespace, "ns", "", "DataImportCron namespace.")
+	flag.StringVar(&cronName, "cron", "", "DataImportCron name.")
+	flag.StringVar(&url, "url", "", "registry source url.")
+	flag.Parse()
+	if url == "" || cronNamespace == "" || cronName == "" {
+		log.Fatalf("One or more mandatory parameters are missing")
+	}
+}
+
+func main() {
+	cmd := "skopeo inspect " + url + " | awk -F'\"' '/Digest/{print $4}'"
```

Comment:
What about support for these params in registry import?

https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/apis/core/v1beta1/types.go#L166-L169

Reply:
How can we use SecretRef and CertConfigMap when we get the image as a ContainerDisk in the server container?